Classes/Notifications/NotificationsViewController.swift

```diff
@@ -56,6 +56,28 @@ FlatCacheListener {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        if NotificationClient.readOnOpen(){
+            self.readOnOpen()
+        }
+        
+    }
+    
+   private func readOnOpen(){
+        var count = 0
+        let showAll = self.showAll
+        for id in notificationIDs {
+            let readOrNot = client.notificationOpened(id: id)
+            if (showAll || !readOrNot) {
+                // count the unread notifications
+                count = count+1
```

Comment:
spaces around `+`

Reply:
I would change the name of `readOrNot` to `isRead`. 
Additionally, you can remove the parenthesis in the `if` clause.

Reply:
`count = count + 1` can be done also like this `count += 1`